client/blocks/reader-post-card/style.scss

```diff
@@ -209,12 +209,6 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 			color: $gray;
 		}
 
-		.reader-post-options-menu {
```

Comment:
@jancavan I _think_ that style should probably continue to live in the reader-post-card, because we always want the ellipsis menu to appear there on `is-compact` cards. For example, on the last example in Devdocs it should be on the top right:

http://calypso.localhost:3000/devdocs/blocks/reader-post-card